magic_example.ipynb

```diff
@@ -13,29 +13,139 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 60,
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Please remove the outputs from the notebook cells before committing.

Reply:
Noted, thanks!